eng/targetingpacks.targets

```diff
@@ -15,8 +15,7 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(DisableImplicitFrameworkReferences)' != 'true' and
-                            '$(TargetFrameworkIdentifier)' == '.NETCoreApp' and
```

Comment:
I know we have no plans of ever introducing a new TargetFramworkIdentifier, but I think it is a good idea to still check that this is `.NETCoreApp` to future proof. Just checking the version might not always be the best approach.